utils/learn-lm.json

```diff
@@ -481,7 +481,7 @@
                     "id": "ballerina-persist",
                     "subDirectories": [
                         {
-                            "dirName": "Overview",
+                            "dirName": "Bal persist overview",
```

Comment:
why we are adding `bal persist` here?

Reply:
That is to update the left nav as `Overview`-> `Bal persist overview`?